File: NeoPlayer/Models/VideoFile.cs
using System.Collections.Generic;

namespace NeoPlayer.Models
{
	public class VideoFile
	{
		[PrimaryKey]
		public int VideoFileID { get; set; }
		public string Identifier { get; set; }
		public string FileName { get; set; }
		[Ignore]
		public string Title { get => Tags[nameof(Title)]; set => Tags[nameof(Title)] = value; }
		[Ignore]
		public string DownloadDate { get => Tags[nameof(DownloadDate)]; set => Tags[nameof(DownloadDate)] = value; }
		[Ignore]
		public string URL { get; set; }
		[Ignore]
		public bool AudioOnly
		{
			get
			{
				if ((!Tags.ContainsKey(nameof(AudioOnly))) || (string.IsNullOrEmpty(Tags[nameof(AudioOnly)])))
					return false;
				if (!bool.TryParse(Tags[nameof(AudioOnly)], out var result))
					return false;
				return result;
			}
			set => Tags[nameof(AudioOnly)] = value.ToString();
		}
		[Ignore]
		public Dictionary<string, string> Tags { get; } = new Dictionary<string, string>();
	}
}

File: NeoRemote/app/src/main/java/neoplayer/neoremote/VideoFile.java
package neoplayer.neoremote;

import java.util.HashMap;

public class VideoFile {
    public int videoFileID;
    public HashMap<String, String> tags = new HashMap<>();

    public VideoFile(int videoFileID, HashMap<String, String> tags) {
        this.videoFileID = videoFileID;
        this.tags = tags;
    }

    public String getTitle() {
        return tags.get("Title");
    }

    public Boolean audioOnly() {
        return Boolean.parseBoolean(tags.get("AudioOnly"));
    }
}